cmake/FindHalide.cmake

```diff
@@ -0,0 +1,2 @@
+message(VERBOSE " Spoofing find_package(Halide) since in-tree builds already have Halide available.")
```

Comment:
Why not instead only look for halide conditionally iff the python_bindings/ is the root cmake dir?